### PR TITLE
Feature/glossary

### PR DESCRIPTION
**Pull request by svengabr**
added new glossary to website

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -1996,5 +1996,141 @@
                 ]
             }
         ]
-    }
+    },
+    "glossary": {
+        "glossaryHeadline": "Glossariung",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:2001]: Or maybe just "Glossar" ?

Comment (Ein-Tim) [reply src/data/faq_de.json:2001]: Yes, please! 😄

Comment (svengabr) [reply src/data/faq_de.json:2001]: Sorry the content is still in redaction, this is just dummy content ;)

Comment (MikeMcC399) [reply src/data/faq_de.json:2001]: Thanks for changing to "Draft" status. That makes it clearer now.

**Pull request merged**